app/components/rest/volume.ts

```diff
@@ -5,11 +5,8 @@ export class VolumeService {
     restFull: restangular.IService;
     static $inject: Array<string> = ['Restangular'];
     constructor(rest: restangular.ICollection) {
-        this.rest = rest.withConfig((RestangularConfigurer) => {
-            RestangularConfigurer.setBaseUrl('/api/v1/gluster');
-        });
+        this.rest = rest;
```

Comment:
This service is specific to gluster and not used anywhere. this file can be removed